ckanext/data_qld/templates/package/snippets/package_form.html

```diff
@@ -1,70 +1,16 @@
-    {# pkg_name used in 3 stage edit #}
-    <input type="hidden" name="pkg_name" value="{{ data.id }}"/>
-
-    {% block errors %}
-        {{ form.errors(error_summary) }}
-    {% endblock %}
-
-    {% block basic_fields %}
-        {% snippet 'package/snippets/package_basic_fields.html', data=data, errors=errors %}
-    {% endblock %}
-
-    {% block metadata_fields %}
-        {% snippet 'package/snippets/package_metadata_fields.html', data=data, errors=errors %}
-    {% endblock %}
-
-    {% block form_actions %}
-        <div class="form-actions">
-            {% block disclaimer %}
-                <p class="action-info small">
-                    {%- trans -%}
-                    The <i>data license</i> you select above only applies to the contents
-                    of any resource files that you add to this dataset. By submitting
-                    this form, you agree to release the <i>metadata</i> values that you
-                    enter into the form under the
-                    <a href="http://opendatacommons.org/licenses/odbl/1-0/">Open Database License</a>.
-                    {%- endtrans -%}
-                </p>
-            {% endblock %}
-            {% block delete_button %}
-            {% if h.check_access('package_delete', {'id': data.id}) and not data.state == 'deleted' and not h.data_qld_data_driven_application(data.data_driven_application)  %}
-                <a id='dataset-delete-button' disabled="disabled"
-                    class="btn btn-danger pull-left"
-                    {% if h.is_ckan_29() %}
-                    href="{{ h.url_for('dataset.delete', id=data.id) }}"
-                    {% else %}
-                    href="{% url_for controller='package', action='delete', id=data.id %}"
-                    {% endif %}
-                    data-module="confirm-action"
-                    data-module-content="{{ _('Deleting this dataset will prevent future use of the dataset’s name. It is highly recommended that dataset issues are rectified rather than deleting and creating a new dataset with the same information.') }}"
-                    data-module-template="{{ h.get_deletion_reason_template() }}">
-                    {% block delete_button_text %}{{ _('Delete') }}{% endblock %}
-                </a>
-                {% set type = 'asset' if h.is_ckan_29() else 'resource' %}
-                {% include "package/snippets/dataset_deletion_" ~ type ~ ".html" %}
-            {% endif %}
-            {% endblock %}
-
-            {% block save_button %}
-                <button class="btn btn-primary" type="submit" name="save">
-                    {% block save_button_text %}
-                        {{ _('Next: Add Data') }}
-                    {% endblock %}
-                </button>
-            {% endblock %}
-
-            {{ form.required_message() }}
-        </div>
-    {% endblock %}
-</form>
+{% ckan_extends %}
+
+{% block delete_button %}
```

Comment:
why such a large change?

Reply:
The comments are the only part we actually need to customize. By inheriting the rest, we make our life simpler.